driver/registry_default_test.go

```diff
@@ -30,10 +30,19 @@ import (
 	"github.com/ory/kratos/selfservice/hook"
```

Comment:
From ~35s to 1.3s (not a typo 😅)

Reason being that `internal.NewFastRegistryWithMocks` was running migrations on every sub-test case, which always took around 0.5s - 1s. Also, parallalizing helped a bit too.

Since we are just testing config here, this should be a good change. LMK, if you think otherwise...